roles/debian-common/defaults/main.yml

```
@@ -0,0 +1,61 @@
+---
+# Compatible with AWS images of Debian and Ubuntu
+debian_common_admin_user: "{{ 'admin' if (ansible_distribution == 'Debian') else 'ubuntu' }}"
+debian_common_regular_user_group: "{{ debian_common_regular_user if (debian_common_regular_user is defined) else '' }}"
+
+# Time zone
+debian_common_ntp_timezone: Europe/Paris
+
+# UFW
+debian_common_firewall: yes
+
+# Common packages
+debian_common_packages:
+  - atop
+  - bash-completion
+  - apt-transport-https
+  - ca-certificates
+  - software-properties-common
+  - curl
+  - tmux
+  - tree
+  - vim
+  - git
+  - htop
+  - acl
+  - dbus
```

I would only install required dependencies and allow to extend it via a configuration variable. WDYT?

```suggestion
  - bash-completion
  - apt-transport-https
  - ca-certificates
  - software-properties-common
  - curl
```